### PR TITLE
Add migrations for client

### LINKED ISSUE #7: Add migrations for client
- add table clients
- add table jobs

### PR DESCRIPTION
**Pull request by levkovalenko**
- add clients table
- add jobs table
- add passports table

close #7 

image

### CODE REVIEW

File: sql/20190409181921_create_clients_table.sql [hunk 1]
```diff
@@ -1,7 +0,0 @@
--- migrate:up
```
Comment (malinink) [inline sql/20190409181921_create_clients_table.sql]: do not remove migration

File: sql/20190424173430_create_clients_table.sql [hunk 2]
```diff
@@ -0,0 +1,14 @@
+-- migrate:up
+CREATE TABLE "clients" (
```
Comment (malinink) [inline sql/20190424173430_create_clients_table.sql]: act with alter table

File: sql/20190424173430_create_clients_table.sql [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    "id"                SERIAL  NOT NULL,
+    "first_name"        TEXT     NOT NULL,
+    "last_name"         TEXT     NOT NULL,
+    "birth_date"        TEXT    NOT NULL,
```
Comment (malinink) [inline sql/20190424173430_create_clients_table.sql]: date?

File: sql/20190424173430_create_clients_table.sql [hunk 4]
```diff
@@ -0,0 +1,14 @@
+-- migrate:up
+CREATE TABLE "clients" (
+    "id"                SERIAL  NOT NULL,
+    "first_name"        TEXT     NOT NULL,
```
Comment (malinink) [inline sql/20190424173430_create_clients_table.sql]: better way is to use varchar(LENGTH) for determining max length

File: sql/20190424173430_create_clients_table.sql [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+    "last_name"         TEXT     NOT NULL,
+    "birth_date"        TEXT    NOT NULL,
+    "sex"               TEXT    NOT NULL,
```
Comment (malinink) [inline sql/20190424173430_create_clients_table.sql]: I suggest to add CHECK on sex field, if it is not enum

File: sql/20190424173430_create_clients_table.sql [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+    "marital_status"    TEXT    NOT NULL,
+    "children"          INT     NOT NULL,
```
Comment (malinink) [inline sql/20190424173430_create_clients_table.sql]: short, check >= 0

File: sql/20190424173430_create_clients_table.sql [hunk 7]
```diff
@@ -0,0 +1,14 @@
+-- migrate:up
+CREATE TABLE "clients" (
+    "id"                SERIAL  NOT NULL,
+    "first_name"        TEXT     NOT NULL,
+    "last_name"         TEXT     NOT NULL,
+    "birth_date"        TEXT    NOT NULL,
+    "sex"               TEXT    NOT NULL,
+    "marital_status"    TEXT    NOT NULL,
```
Comment (malinink) [inline sql/20190424173430_create_clients_table.sql]: also CHECK

File: sql/20190424173819_create_jobs_table.sql [hunk 8]
```diff
@@ -0,0 +1,13 @@
+-- migrate:up
+CREATE TABLE "jobs" (
+    "id"        SERIAL  NOT NULL,
+    "name"      TEXT    NOT NULL,
+    "wage"      INT     NOT NULL,
+    "client_id" INT     NOT NULL,
+    PRIMARY KEY (id),
+    FOREIGN KEY (client_id) REFERENCES clients (id) ON DELETE CASCADE ON UPDATE CASCADE,
+    UNIQUE ("name", "wage", "client_id")
```
Comment (malinink) [inline sql/20190424173819_create_jobs_table.sql]: id must be first in UK, also I suggest dont use such UK. Or, may be it is a good idea to use only (client_d, name) fields for UK

**Review by malinink — CHANGES_REQUESTED** [review]

File: sql/20190409181921_create_clients_table.sql (hunk 1 above)
Comment (levkovalenko) [reply sql/20190409181921_create_clients_table.sql]: but this one wasn't usefull

File: sql/20190424173430_create_clients_table.sql (hunk 4 above)
Comment (levkovalenko) [reply sql/20190424173430_create_clients_table.sql]: I open my practice work for Data Base & Network Technologies and see there TEXT fields everywhere, actually German Krimmel told us to use TEXT fields.

File: sql/20190424173819_create_jobs_table.sql (hunk 8 above)
Comment (levkovalenko) [reply sql/20190424173819_create_jobs_table.sql]: actually person can work at one company on two positions with the different salary

File: sql/20190424173430_create_clients_table.sql (hunk 3 above)
Comment (levkovalenko) [reply sql/20190424173430_create_clients_table.sql]: hmm, may be we can do it, but for prototype we can use string here and in the future change it

**Review by malinink — APPROVED** [review]

**Pull request merged**